lib/stripe/dispute.rb

```diff
@@ -10,7 +10,7 @@ class Dispute < APIResource
     custom_method :close, http_verb: :post
 
     def close(params = {}, opts = {})
-      resp, opts = request(:post, close_url, params, opts)
+      resp, opts = request(:post, resource_url + "/close", params, opts)
```

Comment:
Same comment as yesterday: can you add a comment to the `close_url` method to mark it as deprecated? @brandur-stripe also suggested using `Gem::Deprecate` (e.g. https://github.com/stripe/stripe-ruby/blob/master/lib/stripe/source.rb#L27-L28).

Reply:
I was planning to add this feature to codegen and not change it by hand until it's there. I can do it now if you think it'd be better.